Formula/opam.rb

```diff
@@ -109,12 +36,15 @@ def caveats; <<~EOS
 
     Run the following to initialize your environment variables:
 
-    $  eval `opam config env`
+    $  eval $(opam env)
 
     To export the needed variables every time, add them to your dotfiles.
       * On Bash, add them to `~/.bash_profile`.
       * On Zsh, add them to `~/.zprofile` or `~/.zshrc` instead.
 
+    Starting in 2.0.0 is also possible to set up a shell hook, so this is not
```

Comment:
If this is not needed anymore, we probably want to remove it from the `caveats`.